GNUmakefile

```diff
@@ -0,0 +1,32 @@
+MKDIR_P := mkdir -p
+RM_F := rm -rf
+
+export GO111MODULE=on
+
+Targets := \
```

Comment:
we tend to use all caps or all lowercase for all Makefile variables (be consistent)